src/components/mixins/MultiLineButtonBreakPoints.vue

```diff
@@ -0,0 +1,33 @@
+<script>
+export default {
+  name: "MultiLineButtonBreakPoints",
+  data() {
+    return {
+      viewportWidth: window.innerWidth,
```

Comment:
Wäre es nicht besser, wenn auf die Breite der jeweiligen Komponente geprüft wird anstatt des gesamten Fensters? Wenn dieses Mixin nur in OpenwbBaseButtonGroupInput eingebunden wird, sollte das klappen. Dazu müsste dann ein "ref" auf den umschließenden Tag im #default Template gesetzt werden.